design-system-docs/src/_component_examples/_button/previous_next.erb

```diff
@@ -0,0 +1,19 @@
+---
+title: Previous next
+---
+<div>
+  <%= render CitizensAdviceComponents::Button.new do |c| %>
+    Next
+    <% c.icon_right do
+      render CitizensAdviceComponents::Icons::ArrowRight.new
+    end
+  end %>
+</div>
+<div>
+  <%= render CitizensAdviceComponents::Button.new(variant: :secondary) do |c| %>
+    Previous
+    <% c.icon_left do
+      render CitizensAdviceComponents::Icons::ArrowLeft.new
+    end
+  end %>
+</div>
```

Comment:
I'd be tempted to make these two separate examples but I don't think it matters too much

Reply:
Thanks! I'll leave it as it is for now and merge. Only reason is that I want to see how the content will be changed and I'll need some help with the wording if we split the examples - probably one for @danielnissenbaum